src/util/nodeSelectors.ts

```diff
@@ -200,13 +200,16 @@ export function selectWithLeadingDelimiter(...delimiters: string[]) {
  */
 export function childRangeSelector(
   typesToExclude: string[] = [],
-  typesToInclude: string[] = []
+  typesToInclude: string[] = [],
+  useUnnamedStartAndEnd: boolean = false
```

Comment:
I think I'd go with
```suggestion
  includeUnnamedNodes: boolean = false
```

It's a little confusing, but the whole concept behind this function is a little confusing, and I think this naming is consistent with the idea of this function 

Also we'll be getting rid of it soon so it's not that critical

WDYT @Will-Sommers

Reply:
This function is a kludge that I should go away with `.scm` parsing. I'm not tied to any naming concerns with it, and agree with you Andrew re: included/excluded. If you look at the method closely, it will also include non-specified nodes within a selection, including "excluded" types since we're just returning between 2 inclusion bounds.

Reply:
Yeah haha I ran into that with java since the ';' is included as part of the explicit_constructor_invocation. I think the preferred behaviour might be to stop extending the range once an excluded type is found, but current behaviour is fine for now if we're assuming it's temporary.

Reply:
Would be good to at least add this parameter to doc string

Reply:
Are you using this arg anymore, or was it just for `new`?  If not, I'd just revert this file

Reply:
Reverted.